schema/Contact_Committee.json

```diff
@@ -174,7 +174,7 @@
                 "string",
                 "null"
             ],
-            "minLength": 0,
+            "minLength": 1,
             "maxLength": 9,
```

Comment:
This is a conditional based on country so should be minLength 0 here but minLength 1 in the conditional if section of the schema. Same for the other zip fields in the other contact schemas.

Reply:
This and similar errors were fixed by using the required array in each JSON schema file as the source of truth rather than the fec_spec's REQUIRED value.